### PR TITLE
fixes to Economic Governance abilities

### PR DESCRIPTION
**Pull request by turadg**
## Description

BLD holders must be able to replace a rogue economic committee. https://github.com/Agoric/agoric-sdk/issues/5539 won't be done until MN-1.1 so we need some way for Core Eval to do it in an extreme situation.

### Security Considerations

Creates a `.replaceElectorate` capability for holders of a `GovernedContractFacetAccess`. A group that has access to this already has the ability to vote in a new committee. (Though currently not necessarily the ability to create a new committee as the replacement.) 

### Documentation Considerations

Script demonstrating how its done.

### Testing Considerations

Targeted unit test.
Integration test on local chain, with the dapp-econ-gov UI letting a new committee adjust PSM params and pause APIs.

### CODE REVIEW

File: packages/governance/src/contractGovernor.js [hunk 1]
```diff
@@ -218,6 +218,19 @@ const start = async (zcf, privateArgs) => {
     governedCF,
   );
 
+  /**
+   * @param {Amount<'set'>} newElectorate
+   * @returns {Promise<void>}
+   */
+  const replaceElectorate = newElectorate => {
```
Comment (turadg) [inline packages/governance/src/contractGovernor.js]: needs tests

File: packages/inter-protocol/test/amm/vpool-xyk-amm/test-amm-governance.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ test.before(async t => {
   t.context = { bundleCache };
 });
 
-test('start Economic Committee', async t => {
+test('start Economic Governance', async t => {
```
Comment (turadg) [inline packages/inter-protocol/test/amm/vpool-xyk-amm/test-amm-governance.js]: overzealous find-replace.

File: packages/vats/src/core/utils.js [hunk 3]
```diff
@@ -317,7 +317,7 @@ harden(runModuleBehaviors);
  *
  */
 export const makeAgoricNamesAccess = (
-  log = () => {}, // console.debug
+  log = () => { }, // console.debug
```
Comment (turadg) [inline packages/vats/src/core/utils.js]: my IDE has some Prettier config problem

File: packages/governance/docs/contractGovernance.puml [hunk 4]
```diff
@@ -15,7 +15,7 @@ package "GovernedContract Vat" <<Rectangle>>  {
         +terms: { electionManager, governedParams }
         +getState()
         +getContractGovernor()
-        -getParamManagerRetriever()
+        -getParamMgrRetriever()
```
Comment (Chris-Hibbert) [inline packages/governance/docs/contractGovernance.puml:18]: If you make a change to this file, please run `.../governance/docs/gen.sh`, and checkin the changes to the .png.

File: packages/inter-protocol/test/supports.js [hunk 5]
```diff
@@ -106,7 +106,7 @@ export const installGovernance = (zoe, produce) => {
 };
 
 /**
- * Economic Committee of one.
+ * Economic Governance of one.
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/supports.js]:
```suggestion
 * Economic Committee of one.
```

File: packages/inter-protocol/test/psm/gov-replace-committee.md [hunk 6]
```diff
@@ -0,0 +1 @@
+replace committee
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/psm/gov-replace-committee.md:1]: not very helpful yet

File: packages/inter-protocol/test/psm/gov-replace-committee.js [hunk 7]
```diff
@@ -0,0 +1,244 @@
...
+  committeeName: 'Economic Committee',
+  economicCommitteeAddresses: {
+    // gov1: 'agoric1ldmtatp24qlllgxmrsjzcpe20fvlkp448zcuce',
+    // gov2: 'agoric140dmkrz2e42ergjj7gyvejhzmjzurvqeq82ang',
+    // gov3: 'agoric1w8wktaur4zf8qmmtn3n7x3r0jhsjkjntcm3u6h',
+    gov4: 'agoric109q3uc0xt8aavne94rgd6rfeucavrx924e0ztf',
+  },
+};
+
+// #region Quasi-imports
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/psm/gov-replace-committee.js:17]: extraneous.

File: packages/inter-protocol/test/psm/gov-replace-committee.js [hunk 8]
```diff
@@ -0,0 +1,244 @@
...
+harden(assertPathSegment);
+
+/** @type {(name: string) => string} */
+const sanitizePathSegment = name => {
+  const candidate = name.replace(/[ ,]/g, '_');
+  assertPathSegment(candidate);
+  return candidate;
+};
+
+// #endregion
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/psm/gov-replace-committee.js:154]: Oh, here's the other end.  But I'm still convinced the is leftover copypasta.

File: packages/inter-protocol/test/psm/wildcard-permit.json [hunk 9]
```diff
@@ -0,0 +1 @@
+true
```
Comment (Chris-Hibbert) [inline packages/inter-protocol/test/psm/wildcard-permit.json:1]: unused?

**Review by Chris-Hibbert — APPROVED** [review]
The governance aspects of this look good to me.  I'm less familiar with makefiles and chain updates.

File: packages/inter-protocol/test/psm/gov-replace-committee.js (hunk 7 above)
Comment (dckc) [reply packages/inter-protocol/test/psm/gov-replace-committee.js:17]: I think this is saying: this section is the result of a poor-man's module system where we manually copy the functions we need from elsewhere in the SDK.

File: packages/inter-protocol/test/psm/wildcard-permit.json (hunk 9 above)
Comment (dckc) [reply packages/inter-protocol/test/psm/wildcard-permit.json:1]: it's used in the `Makefile`

**Review by dckc — APPROVED** [review]
looks good.

File: packages/inter-protocol/test/psm/gov-replace-committee.js (hunk 7 above)
Comment (turadg) [reply packages/inter-protocol/test/psm/gov-replace-committee.js:17]: Yes and in IDEs the whole region can be folded
https://code.visualstudio.com/docs/editor/codebasics#_folding
https://www.jetbrains.com/help/rider/Coding_Assistance__Surrounding_with_Region.html#working-with-regions-in-the-file-structure-window

**Pull request merged**